docs/integration/external-integrations/device42-servicenow-connector.mdx

```diff
@@ -302,11 +297,11 @@ Configure the data sync schedules as appropriate for your environment. Search Se
 
 ![](/assets/images/SN_Store_Download_4.png)
 
-These can be scheduled to run at certain times and should be run in the highlighted order: Devices, IPs, Services, Software Details (version 2.0.1), and Software Instances. You can click on an individual schedule to edit it or to **Execute Now**.
+These can be scheduled to run at certain times and should be run in the highlighted order: Affinity Group, Application Components, Business Applications, Database, Database to Instances, D42-Web-Sites, Web Site, Devices, IPs, Services, and Software Instances. You can click on an individual schedule to change running order , period and time. And then you can **Execute Now**.
 
 ![](/assets/images/SN_Store_Download_5.png)
 
-The scheduled imports first run the respective data source to collect and store data in the import set tables for a given data source. After this data has been imported into this table a transform script is run to map the data into ServiceNow's internal tables. Version 3.0.0+ of the ServiceNow integration plugin leverages ServiceNow's IRE module. For more information on this feature, please visit ServiceNow's documentation page:
+The scheduled imports first run the respective data source to collect and store data in the import set tables for a given data source. After this data has been imported into this table a transform script is run to map the data into ServiceNow's internal tables. ServiceNow integration plugin leverages ServiceNow's IRE module. For more information on this feature, please visit ServiceNow's documentation page:
 
 [https://docs.servicenow.com/bundle/utah-servicenow-platform/page/product/configuration-management/concept/c_CMDBIdentifyandReconcile.html](https://docs.servicenow.com/bundle/utah-servicenow-platform/page/product/configuration-management/concept/c_CMDBIdentifyandReconcile.html)
```

Comment:
Replace this URL with:
https://www.servicenow.com/docs/bundle/zurich-servicenow-platform/page/product/configuration-management/concept/c_CMDBIdentifyandReconcile.html